packer/build_image.sh

```diff
@@ -56,7 +56,7 @@ while [ $# -gt 0 ]; do
         "--repo")
             REPO_FOR_INSTALL=$2
```

Comment:
I forgot to commit also change in this line, it is now `REPO_FOR_INSTALL="https://$2"`

i added it directly to next since it was failing next-machine-image